Dockerfile

```diff
@@ -18,9 +18,9 @@ RUN apt-get update && apt-get -y install  unzip \
                         pkg-config \
                         p7zip-full
 
-ENV KERNEL_VERSION  3.16.7
-ENV AUFS_BRANCH     aufs3.16
-ENV AUFS_COMMIT     b3883c3cb86937801fdd2e188032063de617ef73
+ENV KERNEL_VERSION  3.18.1
+ENV AUFS_BRANCH     aufs3.18
```

Comment:
I presumed that `aufs3.18+` will contain changes that would apply to eventually going to be `3.18.2` - whereas this branch would be specific to `3.18.1`